validator/sawtooth_validator/journal/block_store.py

```diff
@@ -77,6 +76,13 @@ def __str__(self):
             out.append(str(value))
         return ','.join(out)
 
+    def set_batch(self, add_pairs, del_keys=None):
```

Comment:
This should be private and should be named some that is not likely to get confused with Sawtooth Batches. perhaps "_commit_to_block_store()"